gossipd/routing.c

```diff
@@ -341,6 +343,12 @@ static void bfg_one_edge(struct node *node, size_t edgenum, double riskfactor)
 	}
 }
 
+/* Determine if the given node_connection is routable */
+static bool nc_is_routable(const struct node_connection *nc, u64 now)
```

Comment:
I would use time_t here; it's u64 where it matters and is self-documenting nicely.